src/libOpenImageIO/imagebuf_test.cpp

```diff
@@ -300,6 +300,25 @@ ImageBuf_test_appbuffer_strided()
             }
         }
     }
+
+    // Test negative strides by filling with yellow, backwards
+    {
+        // Use the ImageBuf constructor from a pointer to the last pixel and
+        // negative strides.
+        ImageBuf neg(ImageSpec(res, res, nchans, TypeFloat),
+                     &mem[res - 1][res - 1][0] /* point to last pixel */,
+                     -nchans * sizeof(float) /* negative x stride */,
+                     -res * nchans * sizeof(float) /* negative y stride*/);
+        const float yellow[nchans] = { 1.0f, 1.0f, 0.0f };
+        ImageBufAlgo::fill(neg, cspan<float>(yellow));
+
```

Comment:
What do you think of adding to the test a guard (say {0.5, 0.5, 0.5}) to check for overflow outside the expected span?

Reply:
Good idea. So amended.